aptos-move/framework/aptos-stdlib/sources/string_utils.move

```diff
@@ -107,7 +107,7 @@ module aptos_std::string_utils {
         native_format_list(&b"a = {} b = {} c = {}", &l);
     }
 
-    #[testonly]
```

Comment:
what's `testonly`? how does it work?

Reply:
This is scary as this can accidentally release a non-test function to mainnet. We should add lint test for unknown annotations

Reply:
Honestly this should be a compiler error, not a lint